src/components/Admin/Admin.js

```diff
@@ -21,7 +21,7 @@ const WATCHLIST_TAB = 1;
 const LOCATIONS_TAB = 2;
```

Comment:
We shouldn't be referencing fluxxor in the component, similar to `Dashboard.js`. Please eradicate Fluxxor from the admin module and propagate data via React `props`.